resources/crds/benchmark_v1alpha1_uperf_cr.yaml

```diff
@@ -10,12 +10,15 @@ spec:
     args:
       hostnetwork: false
       pin: false
-      pin_server: "server_hostname"
-      pin_client: "client_hostname"
+      pin_server: "master-0"
```

Comment:
Any particular reason for `master-0` and `master-1`?  Are they workable default values?

Reply:
Ah, no they are not -- this is my CR I have used for testing :)